static/app/gettingStartedDocs/dotnet/dotnet.tsx

```diff
@@ -29,6 +29,12 @@ Install-Package Sentry -Version ${getPackageVersion(params, 'sentry.dotnet', '3.
 const getInstallSnippetCoreCli = (params: Params) => `
 dotnet add package Sentry -v ${getPackageVersion(params, 'sentry.dotnet', '3.34.0')}`;
 
+const getInstallProfilingSnippetPackageManager = () => `
+Install-Package Sentry.Profiling --prerelease`;
```

Comment:
Is this command right? I copied the one below from docs but there was no equivalent for this one.

Reply:
yes except on iOS and Mac Catalyst where it's not needed because we use the native profiler